sections/appendix/authors.tex

```diff
@@ -0,0 +1,142 @@
+The minimum number of articles that have been written by an author is $1$ and the maximum number is $9906$.
+When we look at the top two most writing authors, we get these:
+\begin{itemize}
+	\item Ove Nørhave (9893 articles)
+	\begin{itemize}
+		\item A well-known journalist, who have been at Nordjyske for over 25 years.
+	\end{itemize}
+	\item System Administrator (9038 articles)
+	\begin{itemize}
+		\item We are not sure why this has been used. It could be a placeholder.
+	\end{itemize}
+\end{itemize}
+
+In \autoref{fig:auhtor_hist_after}, we see that the vast majority of authors have written under $2000$ articles within the three years. 
+All authors and the number of articles they have written can be seen in \autoref{tab:author_table}.
+
+Unlike with Categories, the Author metadata field does not have a natural cut-off point, with a good amount of values in a specific lower range, followed by more evenly distributed numbers.
+Instead, the vast majority of the values fall in a lower range, meaning that setting too high a cut-off point, will result in removing a large portion of the data.
+We instead choose a lower cut-off point, keeping most of the authors, except the ones that contained so few documents, that finding common topics would be inefficient.
+Authors, who have written less than $14$ documents ($0.01\%$ of the number of documents), are removed as part of preprocessing.
+This removes $43$ out of $227$ authors, combining them into a single 'misc' author.
+A total of $204$ documents are assigned to the 'misc' author.
+
+\begin{figure*}[ht]
+	\centering
+	\begin{subfigure}{0.45\textwidth}
+		\centering
+		\includegraphics[width=\linewidth]{figures/author_hist2_before.png}
+		\caption{Before preprocessing}
+		\label{fig:author_hist_before}
+	\end{subfigure}
+	\begin{subfigure}{0.45\textwidth}
+		\centering
+		\includegraphics[width=\linewidth]{figures/author_hist2_14.png}
+		\caption{After preprocessing}
+		\label{fig:auhtor_hist_after}
+	\end{subfigure}
+	\caption{Histogram over amount authors who have written certain number of documents, before and after preprocessing.
+		Authors on the x-axis are grouped into 50 buckets.}
+	\label{fig:author_hist}
+\end{figure*}
+
+\begin{figure*}[ht]
+	\centering
+	\begin{subfigure}{0.45\textwidth}
+		\centering
+		\includegraphics[width=\linewidth]{figures/author_box_before.png}
+		\caption{Before preprocessing}
+		\label{fig:author_box_before}
+	\end{subfigure}
+	\begin{subfigure}{0.45\textwidth}
+		\centering
+		\includegraphics[width=\linewidth]{figures/author_box_14.png}
+		\caption{After preprocessing}
+		\label{fig:auhtor_box_after}
+	\end{subfigure}
+	\caption{Boxplot over the amount of documents written by authors.}
+	\label{fig:author_box}
+\end{figure*}
+
+Other than the 'System Administrator' author, there are a few authors that seem different since they do not have the name of a person.
```

Comment:
```suggestion
Other than the 'System Administrator' author, there are a few other authors that seem different since they do not have the name of a person.
```